### DISCUSSION TITLE
How do I specify and display artificial satellites around the Moon?

### DISCUSSION THREAD

**Discussion by nbhasker** [Stellarium/stellarium#3368] [Q&A] [answered]
NASA Horizons has Moon centric orbital elements for lunar satellites such as Chandrayaan-3. Is there a way to add these to Stellarium and display the lunar satellites?

Thanks

Best Regards

Bhasker

**Comment by alex-w** [comment, marked as answer]
> NASA Horizons has Moon centric orbital elements for lunar satellites such
> as Chandrayaan-3. Is there a way to add these to Stellarium and display the
> lunar satellites?

Why not? But the accuracy of positions will not be very good in a wide
time range IMHO. See ssystem_minor.ini file for example - you need to
define the "moon" of the Moon :)

**Reply by nbhasker** [reply]
Thank you very much. Your suggestion worked. I had a couple of fumbles. First not setting the orbital period which prevented Stellarium from starting up but the log file message was very clear. Then I used the orbital period in seconds (which is what NASA Horizon provides) and the satellite didn't move, I checked the documentation and it appears the orbital period needs to be in (earth) days. I fixed that and got a very plausible animation. I now need to check the RA/Dec against NASA Horizons and see if the orbit is accurate and not just pretty.

Thanks again for your suggestion.

Best regards

Bhasker

Screen capture of the current orbit for today : https://youtu.be/7Y3MhBXj39w

**Reply by alex-w** [reply]
Nice visualization! Please write a record for Chandrayaan-3 here as an example for users.
